docker-compose.yml

```diff
@@ -38,6 +38,8 @@ services:
       - ${PORT__L2GETH_HTTP:-9991}:8545
       - ${PORT__L2GETH_WS:-9992}:8546
     <<: *logging
+    mem_limit: 8g
+    mem_reservation: 6g
```

Comment:
Can we do reserve 6g, no limit? Also please use the docker-compose v3 format: https://docs.docker.com/compose/compose-file/deploy/#resources